### DISCUSSION TITLE
[Script request] VictoriaMetrics

### DISCUSSION THREAD

**Discussion by maxlyth** [community-scripts/ProxmoxVE#516] [Request script] [answered]
### Application Name

VictoriaMetrics

### Website

https://victoriametrics.com

### Description

VictoriaMetrics is a fast and scalable DB for time series data such as that produced by a HomeLab or home automation (ie HomeAssistant). VictoriaMetrics is rapidly gaining ground as a replacement for InfluxDB which is becoming more unwieldy and unapproachable for home users with each release.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by maxlyth** [comment]
I have already built a basic updateable installer that provides dashboard UI and Influx API endpoint suitable for recording instrumentation from HomeAssistant, pfSense etc that can be queried for a dashboard such as Grafana. I have not set up any agents for alerts, backup or downsampling and don't intend to unless there is demand for that.

I also have built a [migration script](https://github.com/maxlyth/influxv2tovm) from InfluxDB v2 that has specific support for HomeAssistant datasets.

I am planning to test this locally for 1-2 weeks before issuing a PR

**Comment by tremor021** [comment, marked as answer]
#2565 Its done here
